stl/inc/format

```diff
@@ -1797,6 +1797,7 @@ public:
 // clang-format off
 template <class _Container>

```

Comment:
Another sidenote: if this specialization was added merely to optimize for `basic_string`, wouldn't it make more sense to get rid of it entirely, then make line 1761 use a trait helper, so we could instead just add a trait for `back_insert_iterator<basic_string>`, reducing code duplication